File: app/src/main/java/com/example/myapplication/ui/bh/BhViewModel.java
package com.example.myapplication.ui.bh;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class BhViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public BhViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is BH fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/example/myapplication/ui/bh/BhViewModel.kt
package com.example.myapplication.ui.bh

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class BhViewModel : ViewModel() {
    private val mText: MutableLiveData<String>

    init {
        mText = MutableLiveData()
        mText.value = "This is BH fragment"
    }

    val text: LiveData<String>
        get() = mText
}